cms/plugin_base.py

```diff
@@ -219,6 +219,22 @@ def get_cache_expiration(self, request, instance, placeholder):
         """
         return None
 
+    def get_vary_cache_on(self, request, instance, placeholder):
+        """
+        Provides hints to the placeholder, and in turn to the page for
+        determining VARY headers for the response.
+
+        Must return one of:
+            - None (default),
+            - String of a case-sensitive header name, or
```

Comment:
Yeah but adding the brackets around it won't really add much vs out overhead of maintaining both ways. Up to you though.